src/wasm-type.h

```diff
@@ -79,6 +79,20 @@ class Type {
 
   // Allows for using Types in switch statements
   constexpr operator uint32_t() const { return id; }
+
+  // Returns the type size in bytes. Only single types are supported.
+  unsigned getByteSize() const;
+
+  // Reinterpret an integer type to a float type with the same size and vice
+  // versa. Only single integer and float types are supported.
+  Type reinterpretType() const;
```

Comment:
This can be just `reinterpret` I think, as that it's on a Type is now implicit?

Reply:
Done